test/escape-as-command-line-opt.json

```diff
@@ -2,7 +2,6 @@
   "machine": "escape-as-command-line-opt",
   "expectations": [
 
-    // Number
```

Comment:
Why the change to json rather than json5?

Reply:
I was trying to figure out why the tests were failing. I'll change it back 👍